<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoCell</class>
 <widget class="QWidget" name="UserInfoCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0,0,0,0);</string>
  </property>
  <widget class="QLabel" name="user_header">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/new/headers/images/headers/friend.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="user_username">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_deleteFriendButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0); border: 1px solid white; border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_ip">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>249</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
